app/main.py

```diff
@@ -14,4 +14,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+
+    for char in phrase:
+        if char.lower() == letter.lower():
+            count += 1
+
+    return count
```

Comment:
You can simplify this.
object ```str``` already has a method ```count()```
check it out
https://docs.python.org/3/library/stdtypes.html#common-sequence-operations